src/pineko/theory.py

```diff
@@ -369,10 +370,9 @@ def fk(self, name, grid_path, tcard, pdf):
         astrong = sc.StrongCoupling.from_dict(tcard)
         # ocard = self.load_operator_card(name)
         # q2_grid = ocard["Q2grid"]
+        #loading ekos
```

Comment:
you're still not running pre-commit :upside_down_face:

Reply:
This time I was sure 😩

Reply:
```
pre-commit install
```
in the repository.

Right after, you can check the content of `.git/hooks/pre-commit`. It should be clear from the code that you installed it (otherwise the file will contain only comments)

Reply:
https://github.com/N3PDF/pineko/pull/27#discussion_r916710466 - don't worry it is just a minor thing, which however is useful

Reply:
Ok thanks this is what I was missing